rdmo/management/tests/test_merge_attributes.py

```diff
@@ -121,16 +124,65 @@ def _create_new_merge_attributes_and_views(db, settings):
     create_copy_of_view_that_uses_new_attribute(db, NEW_MERGE_URI_PREFIXES)
 
 
+@pytest.mark.usefixtures("_create_new_merge_attributes_and_views")
+def test_command_merge_attributes_fails_correctly(db, settings):
```

Comment:
added some test for when it should fail because the input source or target was wrong